src/Components/DetailsPage/DetailsPage.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import "./DetailsPage.css";
+import logo from "../../images/logo.png";
+
+const DetailsPage = () => {
+  return (
+    <div className="details-container">
+      <div className="ba">
+      <div className="pic">
+        <img className="details-image" src={logo} alt="Logo" />
+      </div>
+      <div className="ma">
+        <div className="details-holder">
+          <div>
+            <h2>Amanita parcivolvata</h2>
```

Comment:
Great layout - so when we go to start making the call to this we can interpolate into these fields laid out